packages/html/ds/scripts/build-properties.ts

```diff
@@ -66,7 +66,7 @@ function getSchemaProperties(schema) {
           description,
           type: toType(zodType),
           required,
-          values: zodEnums ? zodEnums : undefined,
+          values: zodEnums || undefined,
```

Comment:
Can `(value as any)._def.values` be 0, "" or null ? Otherwise why is the || needed?

Reply:
good point